sample/DocumentDB-CSharp/run.csx

```diff
@@ -1,6 +1,6 @@
 ﻿using System;
 
-public static object Run(string input, out object newItem)
```

Comment:
While inspecting and cleaning up the other warnings, I saw a warning for this as well in the logs (unused out param).